lighting/raymarch/softShadow.hlsl

```diff
@@ -30,22 +32,25 @@ options:
 #ifndef FNC_RAYMARCH_SOFTSHADOW
 #define FNC_RAYMARCH_SOFTSHADOW
 
-float raymarchSoftShadow( float3 ro, float3 rd, in float mint, in float maxt, float w ) {
+float raymarchSoftShadow(float3 ro, float3 rd) {
+    const float mint = RAYMARCH_SHADOW_MIN_DIST;
+    const float maxt = RAYMARCH_SHADOW_MAX_DIST;
+    const float w = RAYMARCH_SHADOW_SOLID_ANGLE;
+
     float res = 1.0;
     float t = mint;
-    for (int i = 0; i < RAYMARCHSOFTSHADOW_ITERATIONS && t < maxt; i++)
-    {
+    for (int i = 0; i < RAYMARCHSOFTSHADOW_ITERATIONS; i++) {
```

Comment:
Is there a need to do this in HLSL? It's only needed on `PLATFORM_WEBGL`

Reply:
@patriciogonzalezvivo  You're right. But I'm trying to keep both versions as similar as possible. Makes it much easier for me to keep them in sync.